views/forms/BarChartFields.jsx

```diff
@@ -94,7 +100,17 @@ const BarChartFields = ({ classes, formValues, setFormValues, studies }) => {
         fullWidth
       />
       <div className={classes.formLabelRow}>
-        <br />
+        <InputLabel htmlFor='public_checkbox' className={classes.publicText}>
+          Public
+        </InputLabel>
+        <Checkbox
+          checked={formValues.public}
+          onChange={updateFormValues}
+          name='public'
+          color='default'
+          id='public_checkbox'
+          aria-label
+        />
```

Comment:
looks good, doesn't have to be part of this PR, but it may be worth following the pattern `TextField` uses; passing `label` as a prop (ideally required, or with a default -- maybe of `name`) and have `Checkbox` render `InputLabel` and the underlying input. That way inputs always have labels.

Reply:
Thanks for this @WMahoney09

Reply:
💯 🙂 np